website/docs/help-and-support/troubleshooting-guide/deployment-errors.md

```diff
@@ -105,3 +105,14 @@ sudo docker-compose up -d --force-recreate appsmith-internal-server
 ```
 
 In a minute or two, the server should now come up and be ready.
+
+
+## Email errors
+
+If you are using Docker and are having trouble with invitation emails not being sent, even though the Admin email setup is able to send test emails, it may be due to an issue with the configuration of the Docker container.
```

Comment:
I don't think this is a Docker-specific error. It was because the APPSMITH_REPLY_TO and APPSMITH_MAIL_FROM had to have the same email.  This could be same for Kubernetes too.  So I think we should start with 

"If you are having trouble with invitation emails not being sent...... with the configuration of the email parameters"

Reply:
@ankitakinger can you please ocnfirm the above? Thanks :)

Reply:
@danciaclara You're right. This is not docker related error. Sharing the link which explains what was the issue that was causing this:
https://theappsmith.slack.com/archives/C02MUD8DNUR/p1652682757105139?thread_ts=1652681174.350099&cid=C02MUD8DNUR